src/h2olog/misc/gen_raw_tracer.py

```diff
@@ -530,6 +530,7 @@ def build_bpf_header_generator():
     fflush(mem);
     std::string s(buff, buff_len);
     fclose(mem);
+    free(buff);
```

Comment:
Is there a need to use stdio at all?

I think we can either use std::strstream (C++), or use one invocation of asprintf to generate the string at once.

Use of asprintf is fine for h2olog, because it runs only on Linux.

Reply:
That's a good suggestion. If I wrote this from scratch .... I wouldn't use `open_memstream()`. It's easy to make mistakes.

Instead, I just use `std::string here: 678b8abca

[`std::format()`](https://en.cppreference.com/w/cpp/utility/format/format) would be the best, but it's too new (C++20).